src/handlers/miner_hbbft_handler.erl

```diff
@@ -804,6 +844,9 @@ bin_to_msg(<<Bin/binary>>) ->
         {error, truncated}
     end.
 
+sign_skip({proposed_skip, ProposedRound, MyRound}, #state{swarm_keys={_Key, SigFun}}) ->
+    {proposed_skip, ProposedRound, MyRound, SigFun(t2b({proposed_skip, ProposedRound}))}.
```

Comment:
We only sign over the proposed round, not myround, is that ok?

Reply:
I think that this will allow some minor adversarial behavior but I am not sure if we care?  If you're not being honest about your round and want the group to stop, you don't plan on executing anyway?